### PR TITLE
Author style conventions

### PR DESCRIPTION
**Pull request by joelostblom**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,5 +15,7 @@ If you wish to report errata or suggest improvements to wording, please include
 the chapter name in the first line of the body of your report (e.g., `Testing
 Data Analysis`).
 
+Code style conventions can be found in [./style-guide-for-authors.Rmd].
```
Comment (lwjohnst86) [inline CONTRIBUTING.md]: You'll have to move this text based on PR #169

File: style-guide-for-authors.Rmd [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        .method(short_arg)
+        .method(
+            long_arg1,
+            long_arg2))
+    ```
+
+These are not set in stone, just collecting what was suggested in the issue.
+There are more details for what we recommend for learners in [./style.Rmd]
+(Python only so far). Discuss further in [issue
+#116](https://github.com/merely-useful/merely-useful.github.io/issues/116).
```
[the remark this answers is not in the document]
Comment (joelostblom) [reply style-guide-for-authors.Rmd]: This renders fine for me as is, let me know if it is not for you and I can change.

**Comment by joelostblom** [comment]
I changed the file to .md since it is not going in the book anyways.

File: style-guide-for-authors.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (lwjohnst86) [reply style-guide-for-authors.Rmd]: Cool, must be because it doesn't have an empty line above. No worries.

**Pull request merged**